tool/builder3dtools/src/main/java/org/openscience/cdk/modeling/builder3d/TemplateExtractor.java

```diff
@@ -365,8 +365,9 @@ public void makeCanonicalSmileFromRingSystems(String dataFileIn, String dataFile
     public List<IBitFingerprint> makeFingerprintsFromSdf(boolean anyAtom, boolean anyAtomAnyBond,
             Map<String, Integer> timings, BufferedReader fin, int limit) throws Exception {
 
-        IFingerprinter fingerPrinter = new HybridizationFingerprinter(HybridizationFingerprinter.DEFAULT_SIZE,
+        HybridizationFingerprinter fingerPrinter = new HybridizationFingerprinter(HybridizationFingerprinter.DEFAULT_SIZE,
```

Comment:
But only in the 3D builder stack, right? There this was used as a backup: first it would try to find a matching ring system with the correct elements, and second, if no such ring was found, at least a ring system for the same ring sizes but possibly different chem. elements in the ring...